genesis/model/resolver/handler_test.go

```diff
@@ -91,9 +94,11 @@ func TestHandler_GetObject_GlobalScope(t *testing.T) {
 	(*GlobalResolver.globalInstanceMap)["123"] = mockParent
 
 	obj, err := resolverHandler.GetObject(ref, "mockChild")
+	rf := obj.(*mockChild).GetReference()
```

Comment:
`obj.(object.Callable).GetReference()` is better - you bind to common interface, not to specific struct